app/decorators/course_decorator.rb

```diff
@@ -27,6 +27,10 @@ def find_url(provider = object.provider)
     h.search_ui_course_page_url(provider_code: provider.provider_code, course_code: object.course_code)
   end
 
+  def description
+    object.description.to_s.gsub('PGCE with QTS', 'QTS with PGCE')
```

Comment:
does this need to be `gsub` or can it  be `sub`?

Reply:
Can be any of them and will work. I am happy either one.

Reply:
I changed to use sub.